A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -117,6 +117,18 @@ class CDUInitPage {
         Coherent.trigger("AP_VS_VAL_SET", 300);
         Coherent.trigger("AP_HDG_VAL_SET", 180);
     }
+    // Does not refresh page so that other things can be performed first as necessary
+    static async updateTowIfNeeded(mcdu) {
```

Comment:
The UI is entirely single-threaded so `async` will not do anything.

Reply:
So I should also remove await from call `await mcdu.trySetTakeOffWeightLandingWeight(tow.toFixed(1));` as I cannot anymore call it like that from non-async function?

Reply:
Yes.

Reply:
Thanks. Resolved in https://github.com/flybywiresim/a32nx/pull/600/commits/559844ee8f7035558291457dded1d21e84223bff.